examples/viewer.py

```diff
@@ -0,0 +1,152 @@
+from magnum import gl, shaders
+from magnum.platform.glfw import Application
+
+import habitat_sim
+
+settings = {
+    "max_frames": 10,
+    "width": 256,  # Spatial resolution of the observations
+    "height": 256,
+    "scene": "data/scene_datasets/habitat-test-scenes/apartment_1.glb",  # Scene path
+    "default_agent": 0,
+    "sensor_height": 1.5,  # Height of sensors in meters
+    "rgb": True,  # RGB sensor
+    "seed": 1,
+    "enable_physics": True,
+    "physics_config_file": "data/default.phys_scene_config.json",
+    "silent": False,
+    "num_objects": 10,
+    "compute_shortest_path": False,
+    "compute_action_shortest_path": False,
+    "save_png": True,
+    "render_to_ui": True,
+}
+
+navmesh_settings = habitat_sim.NavMeshSettings()
```

Comment:
You need to use if __name__ == __main__. Otherwise, it will make it hard to reuse code.